### PR TITLE
Update code to work with LH 0.8.4

### PR DESCRIPTION
**Pull request by ranjitjhala**
Various updates to various files. These changes are *not backwards compatible* i.e. the new files will not work with older LH versions.

### CODE REVIEW

File: src/Data/VerifiedEq/Instances/Generics.hs [hunk 1]
```diff
@@ -93,18 +99,21 @@ eqPar1Trans eqP eqPTrans x y z
...
 veqPar1 (VerifiedEq eqP eqPRefl eqPSym eqPTrans)
   = VerifiedEq (eqPar1      eqP)
                (eqPar1Refl  eqP eqPRefl)
                (eqPar1Sym   eqP eqPSym)
                (eqPar1Trans eqP eqPTrans)
 
+
+
+
+
```
Comment (RyanGlScott) [inline src/Data/VerifiedEq/Instances/Generics.hs:112]: Are these newlines necessary?

File: src/Data/VerifiedEq/Instances.hs [hunk 2]
```diff
@@ -10,24 +11,32 @@ module Data.VerifiedEq.Instances
...
 import Data.VerifiedEq.Instances.Prod           as X
 import Data.VerifiedEq.Instances.Sum            as X
 
 import Data.VerifiedEq
 import Language.Haskell.Liquid.ProofCombinators
+import Prelude hiding (Either (..))
+
+
+-- imports (Left  _) = ()
+-- imports (Right _) = ()
```
Comment (RyanGlScott) [inline src/Data/VerifiedEq/Instances.hs:30]: What are these comments doing?

File: src/Data/VerifiedEq/Instances/Contra.hs [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+{-@ LIQUID "--ple"        @-}
 
 module Data.VerifiedEq.Instances.Contra (veqContra, eqContra) where
 
 import Data.Functor.Contravariant
 import Data.VerifiedEq
 import Language.Haskell.Liquid.ProofCombinators
 
-{-@ axiomatize eqContra @-}
+{-@ reflect eqContra @-}
```
Comment (RyanGlScott) [inline src/Data/VerifiedEq/Instances/Contra.hs:10]: I'm glad that we're finally calling this thing `reflect` like in the literature :)

File: src/Data/VerifiedEq/Instances/Generics.hs [hunk 4]
```diff
@@ -1,78 +1,84 @@
...
+{-@ reflect eqV1 @-}
 eqV1 :: V1 p -> V1 p -> Bool
 eqV1 _ _ = False
 -- eqV1 x _ = case x of {}
 
 absurd :: V1 p -> a
 absurd x = case x of {}
 
-{-@ eqV1Refl :: x:V1 p -> { eqV1 x x } @-}
+{-@ eqV1Refl :: x:_ -> { eqV1 x x } @-}
```
Comment (RyanGlScott) [inline src/Data/VerifiedEq/Instances/Generics.hs:21]: Is it considered idiomatic LH nowadays style to supply partial type signatures like this? (Honest question—I'm not sure if there's a right answer here or not.)

**Review by RyanGlScott — COMMENTED** [review]
This generally looks good to me. I've left some inline comments, although most (all?) of them might be due to my inexperience with the latest LH developments.

File: src/Data/VerifiedEq/Instances/Generics.hs (hunk 1 above)
Comment (ranjitjhala) [reply src/Data/VerifiedEq/Instances/Generics.hs:112]: No, sorry -- is there any easy way to remove?

File: src/Data/VerifiedEq/Instances.hs (hunk 2 above)
Comment (ranjitjhala) [reply src/Data/VerifiedEq/Instances.hs:30]: again, should be deleted...

File: src/Data/VerifiedEq/Instances/Generics.hs (hunk 4 above)
Comment (ranjitjhala) [reply src/Data/VerifiedEq/Instances/Generics.hs:21]: Yes, both aesthetically -- the other stuff seems redundant, but more importantly, in this case, there's some weirdness with the actual type so `V1 p` was inconsistent with the GHC type. 
(Can't recall what was going on ... but LH gives the suggestion 

```
when this happens, please use '_'
```

and that did the trick here :-)

File: src/Data/VerifiedEq/Instances/Generics.hs (hunk 1 above)
Comment (RyanGlScott) [reply src/Data/VerifiedEq/Instances/Generics.hs:112]: Good to know. I'll do this myself after merging.

**Pull request merged**